demo-shell/src/app.config.json

```diff
@@ -488,6 +488,8 @@
     "adf-version-manager": {
       "allowComments": true,
       "allowDownload": true,
```

Comment:
should be better just check the permissions and give those option enabled and diabled by the permissions?

Reply:
Yeah, but this is more a generic solution. So that if people just want to disable this feature, they can do that even if permissions are fine.